src/components/NavigationDrawer/DrawerContent.tsx

```diff
@@ -46,7 +45,7 @@ export const DrawerContent = () => {
               pathname === href ? 'bg-primaryContainer' : ''
             }`}
           >
-            <Link
+            <a
```

Comment:
We have to use `a` tag instead of `Link` because Next.js `Link` is not working when we add Grafana Observability.

Reply:
CC: @mehrdadmms